packages/maskbook/src/plugins/VCent/define.tsx

```diff
@@ -15,6 +15,7 @@ export const VCentPluginDefine: PluginConfig = {
     postInspector: function Component(): JSX.Element | null {
         const tweetAddress = usePostInfoDetails('postID')
         if (!tweetAddress) return null
+        if (window.location.pathname.split('/')[3] != tweetAddress) return null
```

Comment:
So this is only going to display on the detail page of a tweet? I think it might be a good approach to resolve #2840

Reply:
cc maybe @yisiliu or @guanbinrui?

Reply:
That was the intent of the change.